src/app/view/evident-settings/evident-settings.component.ts

```diff
@@ -54,11 +61,14 @@ export class EvidentSettingsComponent implements OnInit {
             email: data.email || '',
             defaultTradingAccount: data.defaultTradingAccount || '',
             defaultBeneficiaryAccount: data.defaultBeneficiaryAccount || '',
+            frequency: data.frequency || this.issuanceFrequencies[0],
           });
         }
       },
       error: (err) => {
-        this.toastrService.error('Failed to load settings' + err);
+        this.toastrService.warning(
+          'Could not find evident settings, please set them up',
+        );
```

Comment:
This still suggests that evident settings are a must initially, which is not the case. It will be better if you remove it completely